### ISSUE TITLE
Support switching between various languages

### ISSUE THREAD

**Issue by Timoses** [Timoses/Caster#19]
Caster should make it easy to switch between different languages interactively using any engine.

**Comment by LexiconCode** [comment]
I agree we should allow for the configuration engine but this also applies to grammars which goes back to a [framework for configuring specs](https://github.com/Timoses/Caster/issues/10).

**Comment by Timoses** [comment]
I don't see a relation to grammars directly...

This could be as "simple" as
* Run 2 engines in parallel using 2 different models
* One is in pause, one is active
* User has to switch language via GUI (or via a voice command, if a plugin exists to do that)

-> comes down to exposing this functionality through an API. However it may be implemented.
